CHANGELOG.rst

```diff
@@ -1,5 +1,73 @@
 .. _changelog:
 
+0.7.3
+-----
+
+This release contains some very nice improvements to the file and datasets management and visualization as well as project migrations.
+
+New features
+~~~~~~~~~~~~
+
+- **Dataset removal** A dataset can be now removed from a project from the UI. Note that if a removed dataset was part of a project with the Knowledge Graph activated, its metadata will not be removed.
```

Comment:
I'm just thinking if that could be a bit hard to understand for the users. I'd say that from a user perspective the data is gone as they cannot see it anymore anywhere. The project is still KG opted-in so no so sure if we need to say we soft-deleted it.

Reply:
right, I thought the deleted dataset would still be displayed on data search. I'll rephrase this.